pandas/tests/arrays/test_datetimelike.py

```diff
@@ -17,37 +17,40 @@
 
 # TODO: more freq variants
 @pytest.fixture(params=["D", "B", "W", "M", "Q", "Y"])
-def period_index(request):
+def freqstr(request):
```

Comment:
wouldn't object to pushing to pandas/conftest.py itself (but if more clear here okie dokie), followon for sure.